Sources/DockProgress/util.swift

```diff
@@ -55,13 +59,38 @@ final class ProgressCircleShapeLayer: CAShapeLayer {
 	}
 }
 
+extension Int {
+	/**
+	```
+	999 => 999
+	1000 => 1K
+	1100 => 1K
+	2000 => 2K
+	10000 => 9K+
+	```
+	*/
+	var shortStringRepresentation: String {
+		let signum = self.signum()
+		let number = abs(self)
+		if number < 1000 {
+			return "\(self)"
+		} else if number < 10000 {
+			return "\(signum * Int(number/1000))K"
+		} else {
+			return "\(signum * 9)K+"
```

Comment:
I just realized that the `k` should be lowercase. https://en.wikipedia.org/wiki/Kilo-